src/server/game/Maps/Map.cpp

```diff
@@ -646,7 +646,9 @@ bool Map::AddToMap(T* obj)
 
     //something, such as vehicle, needs to be update immediately
     //also, trigger needs to cast spell, if not update, cannot see visual
+    obj->SetItsNewObject(true);
```

Comment:
(removed the `/home/travis/build/TrinityCore/TrinityCore/` part from the error message)
```
src/server/game/Maps/Map.cpp:649:10:
 fatal error: no member named 'SetItsNewObject' in 'Corpse'
    obj->SetItsNewObject(true);
    ~~~  ^
src/server/game/Maps/Map.cpp:3635:32:
 note: in instantiation of function template specialization 'Map::AddToMap<Corpse>' requested here
template TC_GAME_API bool Map::AddToMap(Corpse*);
                               ^
1 error generated.
```